### PR TITLE
Log current time in seconds and microseconds.

### PR DESCRIPTION
**Pull request by arkfps**
Adds timestamps to logging statements so me can relate DRAKVUF-generated events with external ones (e.g., output from other tools).

### CODE REVIEW

File: src/plugins/plugins.h [hunk 1]
```diff
@@ -206,4 +208,18 @@ class drakvuf_plugins
 
 /***************************************************************************/
 
+// Retrieves system time in seconds and microseconds.
+inline timeval get_time()
```
Comment (tklengyel) [inline src/plugins/plugins.h]: `static` perhaps?

Comment (arkfps) [reply src/plugins/plugins.h]: Marking global function `static` makes it inaccessible to other translation units. This makes sense in *.cpp files, but this one is in *.h, i.e., supposed to be included into different translation units. As far as I understand marking it `static` wouldn't have any practical effect.

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please.

**Comment by tklengyel** [comment]
It looks good, two small changes that I would like to request is that in `OUTPUT_CSV` don't include brackets around the timestamp, ie. no `[]`. The other is to keep the plugin name as the first item printed for both output types.

**Comment by arkfps** [comment]
Yep, the brackets are excessive in CSV, in addition the separating commas are missing.

Regarding field order, my thought were that timestamp-first output is easier to scan by eyes, because timestamps are of fixed length contrary to plugin names. Is there practical reason to keep plugin name first?

**Comment by tklengyel** [comment]
Looks good, thanks!

**Pull request merged**